client/python/tox.ini

```diff
@@ -19,10 +19,27 @@ deps =
     poetry
     sphinx
     sphinx-jekyll-builder
+whitelist_externals =
+    cp
 changedir = docs
 commands =
     poetry install -v
     sphinx-build -M jekyll source build
+    cp build/jekyll/python_armada_client.md ../../../docs/python_armada_client.md
```

Comment:
No, actually. We don't know where the root of the repo is in this case except via a relative path -- this could be run from literally anywhere on the system. 

Do you have an idea I'm not thinking of?